docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -64,8 +64,8 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 #### Q: should the number of retries of a job be limited, and if so, where? e.g. a max_retries field in the parent job? wait and see whether people attempt to rerun jobs that have already failed nine zillion times?
 A: Unknown TBD
 
-#### Q: Preventing the same params from being re-run
-A: We have decided to allow it
+#### Q: Preventing the identical params from being re-run within a retry_jobs request
+A: We have decided to allow multiple jobs with the same params to be re-run in the same retry_jobs request.
 
 #### Q: Finding the most recent run of the job: I would very much like to avoid anything involving iterating over a chain of jobs before you can find the most recent run or the original run -- we can come up with better data structures than that!
 A: Unknown TBD, maybe the frontend does it?
```

Comment:
The main concern I had when writing that question was about whether a job with a `retry_parent` could itself be a retry parent. That won't happen, so the bit about the data structures isn't relevant. I'm totally fine with deleting this question, but if it would be of use, it can be edited to be something like

Q: How do we find the most recent retry of a job?

A: The client using the ee2 API would have to figure it out using the `retry_parent` and job creation date fields.

Reply:
ok edited